linera-storage-service/tests/store_test.rs

```diff
@@ -4,10 +4,10 @@
 #![cfg(feature = "storage-service")]
 
 use anyhow::Result;
-use linera_storage_service::client::{StorageServiceStore, StorageServiceStoreInternal};
+use linera_storage_service::client::StorageServiceDatabaseInternal;
 use linera_views::{
     batch::Batch,
-    store::TestKeyValueStore as _,
+    store::TestKeyValueDatabase,
```

Comment:
What is wrong about letting the `as _` which is a little helpful.

Reply:
it worked